python/dgl/dataloading/dataloader.py

```diff
@@ -570,3 +574,190 @@ def __init__(self, graph, indices, graph_sampler, device='cpu', use_ddp=False,
             batch_size=batch_size, drop_last=drop_last, shuffle=shuffle, use_asyncio=use_asyncio,
             use_prefetch_thread=use_prefetch_thread, use_alternate_streams=use_alternate_streams,
             **kwargs)
+
+
```

Comment:
Was the below code meant for dgl/dataloading/pytorch/dataloader.py since it is pytorch specific? It's currently missing the import for `LooseVersion` and `th`.